tests/TailwindBuilderTest.php

```diff
@@ -20,18 +21,25 @@ class TailwindBuilderTest extends TestCase
     protected function setUp(): void
     {
         $fs = new Filesystem();
-        if (file_exists(__DIR__.'/fixtures/var/tailwind')) {
-            $fs->remove(__DIR__.'/fixtures/var/tailwind');
-        }
         $fs->mkdir(__DIR__.'/fixtures/var/tailwind');
     }
 
     protected function tearDown(): void
     {
+        $fs = new Filesystem();
         $finder = new Finder();
         $finder->in(__DIR__.'/fixtures/var/tailwind')->files();
         foreach ($finder as $file) {
-            unlink($file->getRealPath());
+            $fs->remove($file->getRealPath());
```

Comment:
`remove()` can delete non-empty directories iirc.

Reply:
Yeah, good point! And I think it might be a good simplification for that code, I just wanted to last some time on removing those files one by one.. . but I still see that even 1s wait does not help in some builds. I tried to do it in a while loop during 5 seconds, but if it will not help too - I think the best option would be to ignore that folder in windows builds. Let's see if the loop will help